improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -494,10 +489,11 @@ def _probabilities_to_percentiles(
             [x/100.0 for x in percentiles], dtype=np.float32)
 
         forecast_at_percentiles = (
-            np.empty((len(percentiles), probabilities_for_cdf.shape[0]),
-                     dtype=np.float32)
-        )
-        for index in range(probabilities_for_cdf.shape[0]):
+            np.empty((len(percentiles),
+                      probabilities_for_cdf.shape[0]),  # pylint: disable=E1136
+                     dtype=np.float32))
+        for index in range(
+                probabilities_for_cdf.shape[0]):  # pylint: disable=E1136
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'E1136' is used to disable 'unsubscriptable-object' message emission](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4915701)